kibana/README.md

```diff
@@ -96,9 +96,17 @@ In [examples/](./examples) you will find some example configurations. These exam
   ```
 * Setup a port forward and access Kibana at https://localhost:5601
   ```
+  # Setup the port forward
+  kubectl port-forward deployment/helm-kibana-security-kibana 5601
+
+  # Run this in a seperate terminal
   # Get the auto generated password
-  kubectl get secret elastic-credentials -o jsonpath='{.data.password}' | base64 --decode
-  kubectl port-forward deployment/helm-kibana-default-kibana 5601
```

Comment:
kubectl port-forward deployment/helm-kibana-default-kibana 5601 sir I have  a question about that don't you mean by that  :  

 kubectl port-forward deployment/helm-kibana**-security-**kibana 5601

Reply:
Yes, that is exactly what is in the code. I think you are looking at the part of the diff that was deleted rather than the new line which was added below.

Reply:
that's ok  it wasn't updated at the time I looked at it. but still having problem with kibana UI